backend/inspirehep/records/cli.py

```diff
@@ -338,3 +341,85 @@ def remove_bai_from_literature_records(
     for chunk in chunker(documents, 100):
         uuids = [record.meta.id for record in chunk]
         remove_bai_from_literature_authors.delay(uuids)
+
+
+@click.group()
+def legacy_records():
+    """Commands for LegacyRecordsMirror"""
+
+
+@legacy_records.command(help="Export marcxml column from LegacyRecordsMirror table")
+@click.option(
+    "--dir-path",
+    help="Path where the xml files will be saved.",
+    default="/tmp/legacy_xmls",
+    show_default=True,
+)
+@with_appcontext
+def export_xml(dir_path):
+    if not os.path.exists(dir_path):
+        try:
+            os.mkdir(dir_path)
+        except OSError as e:
+            LOGGER.info(f"There was a problem creating the dir: {e}")
+            return
+    try:
+        query = LegacyRecordsMirror.query.all()
```

Comment:
actually, it might be a lot of records, can you use `yield_per` instead of `all`?